src/js/admin/screens/groupsScreen.js

```diff
@@ -0,0 +1,150 @@
+const tableHeader = `
+  <div class="tr groups-table-header">
+    <div class="td sr-img">Image</div>
+    <div class="td sr-number">Number</div>
+    <div class="td sr-id">Key</div>
+    <div class="td sr-name">Name</div>
+    <div class="td sr-edit">Edit</div>
+  </div>
+`;
+
+App.renderGroupsScreen = () => {
+    const groupKeys = Object.keys(App.groups);
+    const header = $(`
+    <div id="cp-header" class="card-header">
+      <div class="cp-name">Groups</div>
+      <div class="cp-control">${groupKeys.length}&nbsp;<i class="material-icons">storage</i></div>
+    </div>
+    `);
+    App.jControlPanelHeader.replaceWith(header);
+    App.jControlPanelHeader = header;
+    const cpBody = $(`
+    <div>
+        <div class="card-header d-flex justify-content-between align-items-center">
+            <h5>Manage groups</h5>
+            <button class="btn btn-primary"><i class="material-icons">import_export</i>&nbsp;Import/Export</button>
+        </div>
+        <div class="card-body">
+        <p class="card-text">Add new group by entering its key</p>
+        <form id="group-addition">
+          <div class="input-group">
+            <input class="form-control invalid" placeholder="Enter new groups’ key" title="Key 1-20 digits">
+            <button class="btn btn-primary btn-raised"><i class="material-icons">add</i>&nbsp;Add new</button>
+          </div>
+        </form>
+      </div>
+        <div id="groups-table" class="table"></div>
+    </div>
+    `);
+    App.jGroupsContainer = cpBody.find('#groups-table');
+    const form = cpBody.find('#group-addition');
+    const input = form.find('input');
+    
+    input.keyup(function(){
```

Comment:
to keyup je dobry, ale v tomhle pripade nemusi byt, chci aby to bylo podobne udelane jako v products, tj. zadam ean kod a submitnu - kdyz ten kod je v katalogu tak vyskoci formular s vyplnenymi udaji, kdyz neni v katalogu tak mi vyskoci formular bez udaju.

kontrola formatu vstupu se da potom udelat pomoci regularnich vyrazu (attribut pattern) - to budeme delat az na konci, kdy budeme mit neoc takovehleho:
```js
App.validation = {
    number: { regex: /\d+/, desc: 'tohle musi byt cislo' },
    phone: { regex: /(+\d{1,3})?\d{3} ?\d{3} ?\d{3}/, desc: 'tohle musi byt telefonni cislo' }
}
```